modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -11,12 +11,12 @@ import * as SelectionBookmark from '../selection/SelectionBookmark';
 import Editor from './Editor';
 
 /**
- * This class enables you to add custom editor commands and it contains
- * overrides for native browser commands to address various bugs and issues.
- *
- * @private
- * @class tinymce.EditorCommands
- */
+  * This class enables you to add custom editor commands and it contains
```

Comment:
Looks like theres some indentation issues with the JSDoc commentts in this file.

Reply:
Thanks, and fixed. VSCode didn't show these changes in the diff